src/config/typeorm/index.ts

```diff
@@ -0,0 +1,27 @@
+export default [
```

Comment:
This is totally fine, but I figured it would be nice to mention it for future - At one point, we won't be able to work with this config (specifically when using continuous deployment). We will need to use connection strings instead.